mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -22,7 +22,7 @@ import RestoreRunModal from './modals/RestoreRunModal';
 import LocalStorageUtils from "../utils/LocalStorageUtils";
 import { ExperimentViewPersistedState } from "../sdk/MlflowLocalStorageMessages";
 
-export const DEFAULT_EXPANDED_VALUE = true;
+export const DEFAULT_EXPANDED_VALUE = false;
```

Comment:
Based on feedback from @mateiz, made nested runs collapsed by default to make it easier to scroll through large numbers of rows